cmake/support_pllgpu.cmake

```
@@ -47,6 +47,7 @@ endmacro()
 
 # Macro to aid in finding cuStateVec lib
 macro(findCustatevec external_libs)
+    set(CQENV "$ENV{CUQUANTUM_SDK}")
```

To keep it consistent with other variables,
 ```suggestion
     set(CUQUANTUM_ENV "$ENV{CUQUANTUM_SDK}")
```